tools/nanopolish/nanopolish_variants.xml

```diff
@@ -0,0 +1,162 @@
+        #end if
+        #if $adv.input_genotype:
+          --genotype '$adv.input_genotype'
+        #end if
+        #if $adv.input_candidates:
+          -c '$adv.input_candidates'
+        #end if
+        #if $adv.input_alt_bc_bam:
+          -a '$adv.input_alt_bc_bam'
+        #end if
+        #if $adv.input_models_fofn:
+          --models-fofn '$input_models_fofn'
+        #end if
+       
+
+
+    ]]></command>
+    <inputs>
+      <!-- index inputs -->
+        <param type="data" name="input_merged" format="fasta,fastq" label="Basecalled merged reads.fa"/>
+        <param type="data" name="input_reads_raw" format="h5,fast5.tar.gz" label="Flat archive file of raw fast5 files"/>
+
+        <!-- variants consensus inputs -->
+        <param type="data" name="input_reads_bam" format="bam" label="Reads aligned to the reference genome" help="(-b)" />
+        <param type="data" name="input_genome_fa" format="fasta" label="The reference genome" help="(-g)"/>
+
+        <section name="adv" title="Optional data inputs">
+            <!-- optional inputs -->
+            <param type="data" name="input_seq_summary" format="txt" optional="true" label="Sequencing summary file from albacore" help="(-s)"/>       
+            <param type="data" name="input_events_bam" format="bam" optional="true" label="Events aligned to the reference genome" help="(-e)" />
+            <param type="data" name="input_genotype" format="vcf" optional="true" label="Call genotypes for the variants in the vcf file" help="(--genotype)" />
+            <param type="data" name="input_candidates" format="vcf" optional="true" 
+                    label="Use variant candidates, rather than discovering them from aligned reads" help="(-c)" />
+            <param type="data" name="input_alt_bc_bam" format="bam" optional="true" label="Alternative basecaller used that does not output event annotations" help="(-a)" />
+            <param type="data" name="input_models_fofn" format="txt" optional="true" label="Read alternative k-mer models" help="(--models-fofn)" />
+       </section>
+
+        <!-- optional params -->
+        <param name="window" type="text" optional="true" label="find variants in window chromsome_name:start-end" help="(-w)"/>
+        <param name="methylation_aware" type="text" optional="true" label="methylation aware polishing and test motifs given" help="(-q)"/>
+        <param name="min_candidate_frequency" type="float" optional="true" value="0.2" label="Extarct if the variant frequency is at least F" help="(-m)"/>
+        <param name="min_candidate_depth" type="integer" optional="true" value="20" label="Extarct if the depth is at least D" help="(-d)"/>
+        <param name="max_haplotypes" type="integer" optional="true" value="1000" label="Consider at most N haplotype combinations" help="(-x)"/>
+        <param name="max_rounds" type="integer" optional="true" value="50" label="Perform N rounds of consensus sequence improvement" help="(--max_rounds)"/>
```

Comment:
You can simply use the `argument="--max_rounds"` attribute here. This will replace the `name` and the `help=` as Galaxy does this for you.